client/client_pool/src/concord_client_pool.cpp

```diff
@@ -414,10 +414,10 @@ void SingleRequestProcessingJob::execute() {
     res = processing_client_->SendRequest(read_config_, std::move(request_));
     reply_size = res.matched_data.size();
     if (callback_) {
-      if (processing_client_->getClientRequestError() != OperationResult::TIMEOUT) {
+      if (processing_client_->getRequestExecutionResult() != OperationResult::TIMEOUT) {
         callback_(bftEngine::SendResult{res});
```

Comment:
- > callback_(res}
We don't pass  SendResult to callback_ anymore; please verify all such places.